_usage/migration.md

```diff
@@ -103,15 +99,15 @@ you can access them with the PostgreSQL protocol as well.  To do that, you can
 use any PostgreSQL client.  Make sure that you connect to the PostgreSQL port
 (by default 5432) and not to the TDS port (by default 1433).
 
-While table and column names will be identical, the names of the schemas in
-PostgreSQL will depend on the
-[migration mode](docs/installation/single-multiple):
+While object names will be identical, the names of the schemas in the migrated SQL
+Server user databases may look different when access through PG, depending on the
+[migration mode_GUC](docs/installation/single-multiple):
 
-- in `single-db` migration mode, the schema names will be the same, no matter
-  if you connect via TDS or via the PostgreSQL protocol
+- in `single-db` migration mode, the schema names of the migrated SQL Server user
+databases will be the same when accessed using T-SQL or using PG SQL
```

Comment:
"PG SQL" is ugly. Should be "the PostgreSQL protocol".
Also, there are two spaces missing - this is a continuation line.